grails-app/services/com/unifina/service/UserService.groovy

```diff
@@ -183,4 +183,8 @@ class UserService {
 			return null
 		}
 	}
+
+	User getUserByUsername(String username) {
```

Comment:
This method doesn't seem to be used anywhere? (Also we shouldn't ever need to look up users by username)

Reply:
Removed the method.